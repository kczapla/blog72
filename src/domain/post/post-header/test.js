import React from 'react'
import renderer from 'react-test-renderer'

import PostHeader from './index'

describe('discrete text link', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<PostHeader post={ { title: 'title' } }/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
