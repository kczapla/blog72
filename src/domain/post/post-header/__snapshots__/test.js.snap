// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`discrete text link renders correctly 1`] = `
<header
  className="sc-jSgupP ilOTjx"
>
  <h2
    className="sc-gsTCUz eZvnGp"
  >
    title
  </h2>
  <div
    className="sc-eCssSg kCCemc"
  >
    <time
      dateTime={"2021-01-01T00:00:00.000Z"}
    >
      Jan 01, 2021
    </time>
    <span
      className="sc-bdfBwQ kagGIA"
    >
       by 
    </span>
    <span
      className="sc-dlfnbm hahudT"
    >
      <a
        className="sc-hKgILt blFMmE"
        href="href"
      >
        name
      </a>
    </span>
  </div>
</header>
`;
